### PR TITLE
[TAS-1140] 👔 Merge /store & /store/books into one page

### PR DESCRIPTION
**Pull request by nwingt**
- [x] Scroll position reset
- [x] Filter out highlighted items in featured tab

### CODE REVIEW

File: src/pages/store.vue [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+import walletMixin from '~/mixins/wallet';
+
+export default {
+  name: 'StoreIndexPage',
+  mixins: [inAppMixin, navigationListenerMixin, walletMixin, bookstoreMixin],
+  layout: 'default',
+  async fetch({ route, redirect, localeLocation, store }) {
+    if (checkIsForcedInAppPage(route)) {
+      redirect(301, localeLocation({ name: 'store-articles' }));
+    }
```
Comment (williamchong) [inline src/pages/store.vue:94]:
```suggestion
      redirect(301, localeLocation({ name: 'store-articles' }));
      return;
    }
```

File: src/pages/store.vue [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+        },
+        {
+          text: this.$t('store_books_page_tab_free'),
+          value: 'free',
+        },
+      ];
+    },
+  },
+  mounted() {
+    console.log('mounted', this.$route.name);
```
Comment (williamchong) [inline src/pages/store.vue:225]:
```suggestion
```

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/pages/store.vue (hunk 2 above)
Comment (nwingt) [reply src/pages/store.vue]: Removed

File: src/pages/store.vue (hunk 1 above)
Comment (nwingt) [reply src/pages/store.vue]: Fixed

File: src/pages/store/index.vue [hunk 3]
```diff
@@ -1,235 +1,7 @@
...
-      logTrackerEvent(
-        this,
-        'nft_book_featured',
-        'nft_book_shelf_click_avatar',
-        classId,
-        1
-      );
-    },
+  fetch({ redirect, localeLocation }) {
+    redirect(301, localeLocation({ name: 'store-featured' }));
```
Comment (williamchong) [inline src/pages/store/index.vue:4]:
```suggestion
    redirect(302, localeLocation({ name: 'store-featured' }));
```

**Review by williamchong — CHANGES_REQUESTED** [review]

**Pull request merged**